tox.ini

```diff
@@ -96,6 +96,7 @@ deps = pytest==4.6.9
        pytest-cov==2.8.1
        pytest-xdist==1.31.0
        mock==3.0.5
+       attrs==20.3.0
```

Comment:
`attrs!=21.1`, would this work?

Reply:
Possibly, it would then fall back to pip to figure out the rest out, but this is only in `[testenv:py34]`, so pinning it makes more sense since it's always going to be that version.

Reply:
Oh, indeed, I missed the `[testenv:py34]` context.
Looks reasonable, then.